### DISCUSSION TITLE
No scrolling possible with Aldi LIGHTWAY smart home LED-lamp - candle (light)

### DISCUSSION THREAD

**Discussion by tdse13** [Koenkk/zigbee2mqtt#7645] [Devices]
Actually, I have 3 Aldi bulbs running in Home Assistant. Unfortunately, no scrolling is possible. As a result only part of the information is visible. Is this a zigbee2mqtt issue? Or a Home Assistant issue? Could somebody fix it. Thank you.

<img width="417" alt="Screen Shot 2021-05-30 um 09 10 11" src="https://user-images.githubusercontent.com/45556549/120109207-bad1c000-c168-11eb-820b-556ac496d66b.png">
<img width="460" alt="Screen Shot 2021-05-30 um 09 10 23" src="https://user-images.githubusercontent.com/45556549/120109210-bc02ed00-c168-11eb-8c7e-12c6c38f8311.png">

**Comment by Koenkk** [comment]
This is a HA issue.
